# goal
sensor_msgs/Image image
---
# result
int64 number_of_people
mdr_perception_msgs/FaceBoundingBox[] bounding_boxes
string[] emotions
bool success
---
# feedback
string current_state
string message
